src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
```

Comment:
This line of code is too long. Remember that lines of code should have 80 characters max. It is easier to read shorter lines.